### PR TITLE
Don't add noop after indentation

### LINKED ISSUE #424: Fix noop to the right of indentation

### PR DESCRIPTION
**Pull request by m09**
Fixes #424

### CODE REVIEW

File: lookout/style/format/feature_extractor.py [hunk 1]
```diff
@@ -594,7 +594,7 @@ def _add_noops(self, vnodes: Sequence[VirtualNode], path: str, index_labels: boo
                                                 end=Position(0, 1, 1), y=noop_label, path=path))
         for vnode, next_vnode in zip(vnodes, islice(vnodes, 1, None)):
             augmented_vnodes.append(vnode)
-            if vnode.y is None and next_vnode.y is None:
+            if vnode.y is None and not vnode.is_accumulated_indentation and next_vnode.y is None:
```
Comment (vmarkovtsev) [inline lookout/style/format/feature_extractor.py]: What if it is accumulated dedentation?

Comment (m09) [reply lookout/style/format/feature_extractor.py]: sorry I misread your comment. dedentation is also tagged as accumulated_indentation

**Comment by vmarkovtsev** [comment]
So what's our plan here? Let @zurk change the way he tests?

**Comment by m09** [comment]
That's what we discussed yesterday yes

**Pull request merged**